components/scream/tests/uncoupled/spa/spa_stand_alone.cpp

```diff
@@ -7,6 +7,9 @@
 #include "share/grid/mesh_free_grids_manager.hpp"
 #include "share/atm_process/atmosphere_process.hpp"
 
+#include "physics/rrtmgp/atmosphere_radiation.hpp"
```

Comment:
Why does SPA stand alone need rrtmgp and YAKL?

Reply:
Removed.